### PR TITLE
Smuggling Tweaks

### PR DESCRIPTION
**Pull request by Cheackraze**
## About the PR

Tweaks many things about the smuggling system. Makes the static posters have a random chance of being dead drop locations. Adds a low/medium difficulty challenge to the dead drops.  Slightly reduces chance any given crate is a contraband crate as a result, thereby lowering payouts on average. And finally, increases the potential cooldown time from 45 minutes to 90 minutes. The minimum is still 15 minutes, so it should still have a much longer average.
## Technical details

Dead drops may be added to a few more POI in the future due to their modified spawn parameters.
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: The Syndicate have increased the defenses on their smuggling drops.
- tweak: Changed the spawn rates and percentages of dead drops.

### CODE REVIEW

**Comment by dvir001** [comment]
I hate YAML

File: Resources/Maps/deaddrop.yml [hunk 1]
```diff
@@ -251,6 +289,27 @@ entities:
     - pos: 1.5,1.5
       parent: 1
       type: Transform
+- proto: DebugAPC
```
Comment (FoxxoTrystan) [inline Resources/Maps/deaddrop.yml:292]: DebugAPC... ehhh...
Are you sure?

Comment (Cheackraze) [reply Resources/Maps/deaddrop.yml:292]: was a mistake but also I cant find what makes the debug apc so special anyhow. the 200kw one is way stronger anyhow. But yea, ill make sure it gets changed

**Review by dvir001 — APPROVED** [review]

File: Resources/Maps/deaddrop.yml (hunk 1 above)
Comment (dvir001) [reply Resources/Maps/deaddrop.yml:292]: When you break it you get normal apc, not an issue.

**Pull request merged**